src/ert/parsing/config_schema_deprecations.py

```diff
@@ -0,0 +1,149 @@
+from .deprecation_info import DeprecationInfo
+
+REPLACE_WITH_GEN_KW = [
+    "RELPERM",
+    "MULTZ",
+    "EQUIL",
+    "GEN_PARAM",
+    "MULTFLT",
+]
+JUST_REMOVE_KEYWORDS = [
+    "UMASK",
+    "LOG_FILE",
+    "LOG_LEVEL",
+    "ENKF_RERUN",
```

Comment:
You omit `GEN_KW_TAG_FORMAT`, is that on purpose?

Reply:
Oops!! Not on purpose, I put it back